src/javascript/app/ng-wig/ng-wig.component.spec.js

```diff
@@ -253,6 +261,32 @@ describe('component: ngWig', () => {
                 expect(controller.ngModelController.$setViewValue).toHaveBeenCalledWith('<p></p>');
             });
 
+            it('should render an empty element', function(){
+              element = getCompiledElement('<ng-wig ng-model="text1" placeholder="My placeholder"><ng-wig>');
+              ngWigElement = angular.element(element[0].querySelector('#ng-wig-editable'));
+              expect(ngWigElement.html()).toEqual('');
+            });
+
+            it('should render the model value even if a placeholder exits', function(){
+              scope.string = "This is a value";
+              let el = `<ng-wig ng-model="string" placeholder="My placeholder"><ng-wig>`;
+              element = getCompiledElement(el);
+              ngWigElement = angular.element(element[0].querySelector('#ng-wig-editable'));
+              expect(ngWigElement.html()).toEqual('This is a value');
+            });
+
+            it('should render the model value even if a placeholder exits and the model value is recevied async', function(){
+              let el = `<ng-wig ng-model="string" placeholder="My placeholder"><ng-wig>`;
+              element = getCompiledElement(el);
+              ngWigElement = angular.element(element[0].querySelector('#ng-wig-editable'));
+
+              $timeout(function () {
+                scope.string = "This is a value";
+                expect(ngWigElement.html()).toEqual('This is a value');
+              }, 100);
+            });
```

Comment:
I don't think so. We're not testing a method using $timeout. We're using it to test an async process. But I'll add it.